source/server/listener_manager_impl.cc

```diff
@@ -289,6 +304,16 @@ bool ListenerManagerImpl::addOrUpdateListener(const Json::Object& json) {
   return true;
 }
 
+bool ListenerManagerImpl::hasListenerWithAddress(const ListenerList& list,
+                                                 const Network::Address::Instance& address) {
+  for (const auto& listener : list) {
```

Comment:
This also goes away I think in v2, as we no longer do the explicit bind/no-bind thing at the Listener level.